### ISSUE TITLE
Bring psidR back to CRAN?

### ISSUE THREAD

**Issue by zeileis** [floswald/psidR#61]
Florian @floswald, as co-maintainer of the [Econometrics CRAN task view](https://CRAN.R-project.org/view=Econometrics) I noticed that `psidR` was recently archived on CRAN for internet access policy violations. Do you plan on updating the package and bringing it back to CRAN? Can I help with anything?

Thanks in advance for your work on the package & best wishes, Achim

**Comment by floswald** [comment]
hey @zeileis 
whoops yes indeed, I completely forgot about that. I do plan to bring it back. do I need to do anything else apart from re-submitting it? thanks

**Comment by zeileis** [comment]
Usually, the crucial point with internet access from the [CRAN Policy](https://cran.r-project.org/web/packages/policies.html#Source-packages-1) is:

> Packages which use Internet resources should fail gracefully with an informative message if the resource is not available or has changed (and not give a check warning nor error).

**I guess** (haven't looked at the details) that when you open URLs via `openxlsx::read.xlsx()`, then it might lead to an error if `download.file()` does not work. This again will probably lead to an error in `R CMD check`.

So before resubmitting you should make sure that the checks work successfully also if you are offline.

**Comment by floswald** [comment]
yes indeed - it did not help that my package tests were running on Uwe Ligge's server on the day that the internet stopped working (crowdstrike incident). I have turned off that particular test now, not related to package functionality anyway. So, I just resubmit as new version, that's it?

**Comment by zeileis** [comment]
Florian, just wanted to check whether you have had the chance to look at this (prompted by https://github.com/cran-task-views/Econometrics/issues/28). Let me know if I can help with anything.

**Comment by zeileis** [comment]
I just noticed that the package is back on CRAN now. Thanks, Florian!
